Documentation/sources/content-card-ui/public-classes/uievent.md

```diff
@@ -0,0 +1,39 @@
+# Sealed Class - UIEvent
```

Comment:
Looks like the content here is wrong. Were you intending to add docs for `UIAction` here?